source/beamline/CLS/CLSSIS3820ScalerContinuousMode.h

```diff
@@ -1,31 +1,25 @@
-#ifndef CLSSIS3820SCALERMODECONTROL_H
-#define CLSSIS3820SCALERMODECONTROL_H
+#ifndef CLSSIS3820SCALERCONTINUOUSMODE_H
+#define CLSSIS3820SCALERCONTINUOUSMODE_H
 
-#include "beamline/AMPseudoMotorControl.h"
+#include "beamline/AMEnumeratedControl.h"
 
-class CLSSIS3820ScalerModeControl : public AMPseudoMotorControl
+class CLSSIS3820Scaler;
+
+class CLSSIS3820ScalerContinuousMode : public AMEnumeratedControl
```

Comment:
AcquisitionMode works for me! I'll make the change, for both the class name (CLSSIS3820ScalerContinuousMode -> AcquisitionMode) and the enum name.